mypy_django_plugin/lib/helpers.py

```diff
@@ -369,9 +373,20 @@ def bind_or_analyze_type(t: MypyType, api: SemanticAnalyzer, module_name: Option
     That should hopefully give a bound type."""
     if isinstance(t, UnboundType) and module_name is not None:
         node = api.lookup_fully_qualified_or_none(module_name + "." + t.name)
-        if node is not None and node.type is not None:
-            return node.type
-
+        if node is not None:
```

Comment:
Can we rename `node` to `sym` or `symbol` instead?

```suggestion
        if symbol is not None:
```